pathOr.ts

```diff
@@ -0,0 +1,18 @@
+import { Curry3, Obj } from "./utils/types.ts"
+import curryN from "./utils/curry_n.ts"
+import { defaultTo } from './defaultTo.ts'
+import { path } from './path.ts'
+
+function _pathOr(d: any, p: Array<any>, obj: any) {
```

Comment:
Change type of path to `Path` declared in `paths.ts`.
Also change type of obj to `ObjRec`